handsontable/src/core.js

```diff
@@ -2546,7 +2546,7 @@ export default function Core(rootElement, userSettings, rootInstanceSymbol = fal
       });
     }
 
-    instance.runHooks('afterCellMetaReset');
+    instance.runHooks('afterCellMetaReset', 'updateSettings');
```

Comment:
This hook runs only once when the `updateSettings` is called and only here. I don't see the point of adding a new argument to the method because there would be nothing to compare it to. The `source` would always be the same.

Reply:
Good point, I'll remove it

Reply:
Good point. I changed it